src/createTranslator.js

```diff
@@ -22,6 +22,10 @@ const createTranslator = (keys) => {
           return render(translation.join("\n"), params)
         }
       }
+      else if (!(/string|number|boolean/).test(typeof translation)) {
```

Comment:
I'd rather like it put on top, before the `Array.isArray(translation)` test, and test it with equality operators, not though a regex test